PostHog/Replay/UIImage+Util.swift

```diff
@@ -11,14 +11,19 @@
 
     extension UIImage {
         func toBase64(_ compressionQuality: CGFloat = 0.3) -> String? {
-            let jpegData = jpegData(compressionQuality: compressionQuality)
-            let base64 = jpegData?.base64EncodedString()
+            toWebPBase64(compressionQuality) ?? toJpegBase64(compressionQuality)
```

Comment:
will `toWebPBase64` return nil here if it fails or still returns the string `data:image/webp;base64,` without the encoded image? otherwise the fall back will never work

Reply:
ah ok, missed the webpData bits here.